src/postprocess.ts

```diff
@@ -65,9 +65,7 @@ export const getPostProcessHook = (
     return src => src; // Identity function
   }
 
-  const plugins = tsJestConfig.babelConfig
-    ? tsJestConfig.babelConfig.plugins || []
-    : [];
+  const plugins = Array.from(tsJestConfig.babelConfig && tsJestConfig.babelConfig.plugins || []);
```

Comment:
Is this different from the original implementation in any way?

Reply:
I believe this creates a new array each time, rather than re-using the existing one, which was the source of the big (same array, push to it every time)

Reply:
Yes, creating a new array here solves a performance problem.

Without this change, the `doesn't accumulate commonjs module transforms on consecutive calls` test would fail because the `plugins` array would contain a duplicate transform.

In my app, `transform-es2015-modules-commonjs` was being applied 150+ times to 100+ files.  That makes `ts-jest` very slow.

Reply:
Thanks for the explanation. I see your point now!